airflow/providers/amazon/aws/example_dags/example_appflow.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.providers.amazon.aws.operators.appflow import (
+    AppflowRecordsShortCircuit,
+    AppflowRunAfterOperator,
+    AppflowRunBeforeOperator,
+    AppflowRunDailyOperator,
+    AppflowRunFullOperator,
+    AppflowRunOperator,
+)
+
+SOURCE_NAME = "salesforce"
+FLOW_NAME = "salesforce-campaign"
+
+with DAG(
+    "example_appflow",
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
+    catchup=False,
+    tags=["example"],
+) as dag:
+
+    # [START howto_appflow_run]
+    run = AppflowRunOperator(
+        task_id="campaign-dump",
```

Comment:
For all tasks in the example DAGs, the value of task_id should match.  For example, please use either:
```
run = AppflowRunOperator(
        task_id='run'
```

or
```
campaign_dump = AppflowRunOperator(
        task_id='campaign_dump'
```

Reply:
Done!

Reply:
Nitpick:  Missed a spot.  Looks like you got all of them except L45.

Reply:
Oh good catch! 👏 

I've just fixed at https://github.com/apache/airflow/pull/24057/commits/aaeeaaa072bea04b198dbecf5f9d88e300d6dcef